x-pack/plugins/index_management/public/application/services/api.ts

```diff
@@ -311,3 +312,28 @@ export function useLoadNodesPlugins() {
     method: 'get',
   });
 }
+
+export const useLoadEnrichPolicies = () => {
+  return useRequest({
+    path: `${INTERNAL_API_BASE_PATH}/enrich_policies`,
+    method: 'get',
+  });
+};
+
+export async function deleteEnrichPolicy(policyName: string) {
+  const result = sendRequest({
+    path: `${API_BASE_PATH}/enrich_policies/${policyName}`,
```

Comment:
I think we should use the internal path for the delete and execute routes too, wdyt?

Reply:
I think we can do it for these other two just to keep it consistent with the list enrich policies method, but I think this sort of changes we should address in a more cohesive way through the plugin as now we will have certain routes that are "internal" and others that are public. I can create an issue and perhaps we can discuss that during our next sync!